easybuild/easyconfigs/g/GROMACS/README.md

```diff
@@ -104,11 +108,20 @@ engines.
     of enabling support for the `cray-python` modules. Therefore the corresponding 
     EasyConfigs of GROMACS have also been replaced.**
 
+### GROMACS 2023.2 with AMD GPU support for CPE 22.12
+
+  * There are multiple easyconfig files for running GROMACS with AMD GPU acceleration, that follows [installation guide](https://manual.gromacs.org/2023.2/install-guide/index.html#sycl-gpu-acceleration-for-amd-gpus).
+
+    * [HeFFTe enabled](https://manual.gromacs.org/2023.2/install-guide/index.html#using-heffte) version for multi-node, multi-GPU execution with MPI and hipSYCL
+    * Generic hipSYCL version with MPI and GPU support enabled
```

Comment:
This needs clarification, "generic" is too vague. This is the same as the above just without HeFFTe; it might be advantageous running single-node but may not be suitable for multi-node runs (with PME).